dll/win32/shell32/CDefView.cpp

```diff
@@ -97,7 +97,7 @@ struct MenuCleanup
 class CDefView :
     public CWindowImpl<CDefView, CWindow, CControlWinTraits>,
     public CComObjectRootEx<CComMultiThreadModelNoCS>,
-    public IShellView2,
+    public IShellView3,
```

Comment:
This change does not seem related to `STDMETHOD` changes...

Reply:
I had embarrassed with deciding whether `CDefView::CreateViewWindow3` has to override. If it was overrided, then the class has to inherit `IShellView3`.